libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -889,10 +890,10 @@ bool ASTJsonExporter::visit(BinaryOperation const& _node)
 
 bool ASTJsonExporter::visit(FunctionCall const& _node)
 {
-	Json::Value names(Json::arrayValue);
+	Json names(Json::array());
```

Comment:
Just as a remark - you're a bit inconsistent between using ``Json names(Json::array());`` and ``Json names = Json::array();`` :-). But I'm gonna ignore that :-).

Reply:
You're right, I will try to do that more consistently.